### PR TITLE
[HAL-1822] TopologyTasks.RunningServers uses invalid resource addresses

### PR DESCRIPTION
**Pull request by anilabhabaral**
https://issues.redhat.com/browse/HAL-1822

### CODE REVIEW

File: core/src/main/java/org/jboss/hal/core/runtime/TopologyTasks.java [hunk 1]
```diff
@@ -95,7 +95,7 @@ public final class TopologyTasks {
     public static final String HOSTS = "topologyTasks.hosts"; // List<Host>
     public static final String SERVER_GROUPS = "topologyTasks.serverGroups"; // List<ServerGroup>
     public static final String SERVERS = "topologyTasks.servers"; // List<Server>
-    public static final String SERVER = "topologyTasks.server"; // Server
```
Comment (hpehl) [inline core/src/main/java/org/jboss/hal/core/runtime/TopologyTasks.java:98]: Please leave the constant unchanged.

Comment (hpehl) [reply core/src/main/java/org/jboss/hal/core/runtime/TopologyTasks.java:98]: And change the code in line [664](https://github.com/hal/console/blob/main/core/src/main/java/org/jboss/hal/core/runtime/TopologyTasks.java#L664) from:

```java
ResourceAddress address = new ResourceAddress()
        .add(ModelDescriptionConstants.HOST, host)
        .add(SERVER, WILDCARD);
``` 

to:

```
ResourceAddress address = new ResourceAddress()
        .add(ModelDescriptionConstants.HOST, host)
        .add(ModelDescriptionConstants.SERVER, WILDCARD);
```

**Review by hpehl — CHANGES_REQUESTED** [review]

**Review by hpehl — APPROVED** [review]

**Comment by hpehl** [comment]
Thanks for the fix @anilabhabaral

**Comment by anilabhabaral** [comment]
Thank you @hpehl

**Pull request merged**